starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,15 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // DONE TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
+  articleController.initPage = function () {
+    $('.tab-content').hide();
+    $('#articles').show();
+  }
 
+  app.Article.fetchAll();
```

Comment:
Also, fetchAll needs a callback. Make sure to focus on this part of the TODO:
 DONE TODO: Setup a function that kicks off the fetching and rendering of articles, using the same code that used to be in index.html